test/sql/date/test_timestamp.cpp

```diff
@@ -134,3 +134,40 @@ TEST_CASE("Test storage for timestamp type", "[timestamp]") {
 	}
 	DeleteDatabase(storage_database);
 }
+
+TEST_CASE("Test timestamp functions", "[timestamp]") {
+	unique_ptr<QueryResult> result;
+	DuckDB db(nullptr);
+	Connection con(db);
+
+	result = con.Query("SELECT AGE(TIMESTAMP '1957-06-13');");
```

Comment:
This fails very rarely at midnight exactly. You don't need to fix this, just something to be aware of. If you do want to fix this I would get the current_timestamp as part of the result instead of getting it separately from the Timestamp api, e.g.:
```sql
SELECT NOW(), AGE(TIMESTAMP '1957-06-13');
```